dev-test/star-wars/types.ts

```diff
@@ -248,6 +248,24 @@ export type CreateReviewForEpisodeMutation = {
   createReview?: { __typename?: 'Review'; stars: number; commentary?: string | null } | null;
 };
 
+export type ExcludeQueryAlphaQueryVariables = Exact<{
+  episode?: InputMaybe<Episode>;
+}>;
+
+export type ExcludeQueryAlphaQuery = {
+  __typename?: 'Query';
+  hero?: { __typename?: 'Droid'; name: string } | { __typename?: 'Human'; name: string } | null;
+};
+
+export type ExcludeQueryBetaQueryVariables = Exact<{
+  episode?: InputMaybe<Episode>;
+}>;
+
+export type ExcludeQueryBetaQuery = {
+  __typename?: 'Query';
+  hero?: { __typename?: 'Droid'; name: string } | { __typename?: 'Human'; name: string } | null;
+};
+
```

Comment:
The changes to multiple `star-wars` files are nothing to be alarmed about. They are just due to adding the `ExcludeQueryAlpha` and `ExcludeQueryBeta` queries, which I added because they are contra-variant in terms of their inclusion or exclusion in `types.excludeQueryAlpha.ts` and `types.excludeQueryBeta.ts`; see commit messages for details. (Although I realized this doesn't actually test the contravariance during watch mode, unless you manually edit `codegen.ts` to remove one of the targets, so maybe they could be removed - but they're harmless, and at least this is testing negation contravariance between those two output targets during build mode.)